zerver/lib/actions.py

```diff
@@ -1620,15 +1620,11 @@ def should_send(setting: str, row: Dict[str, Any]) -> bool:
 
         if possible_wildcard_mention:
             # If there's a possible wildcard mention, we need to
-            # determine which users would receive a wildcard mention
-            # notification for this message should the message indeed
-            # contain a wildcard mention.
-            #
-            # We don't have separate values for push/email
-            # notifications here; at this stage, we're just
-            # determining whether this wildcard mention should be
-            # treated as a mention (and follow the user's mention
-            # notification preferences) or a normal message.
+            # determine the set users have enabled the "wildcard
```

Comment:
```suggestion
            # determine the set of users who have enabled the "wildcard
```

Reply:
I believe Tim fixed this while merging acdce4df4797171fbcd807e0d39fc94d48e475dd; marking this as resolved. Thanks!